packages/app/src/elements/toggle-button.ts

```diff
@@ -13,7 +13,7 @@ export class ToggleButton extends LitElement {
     reverse: boolean = false;
 
     @property()
-    label: string | TemplateResult = "";
+    label: string | TemplateResult | TemplateResult<1> = "";
```

Comment:
This just avoided a linting error in the IDE.

Reply:
Honestly I'd rather figure out **why** the type checker was complaining and fix the underlying problem instead. There should be only one type definition for `TemplateResult`. If there are more than one then something is off.